account_followup/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Payment Follow-up Management
+============================
+
+Module to automate letters for unpaid invoices, with multi-level recalls.
+
+Usage
+=====
+
+You can define your multiple levels of recall through the menu:
+---------------------------------------------------------------
+    Accounting > Configuration > Follow-up > Follow-up Levels
+
+Once it is defined, you can automatically print recalls every day through
+-------------------------------------------------------------------------
+simply clicking on the menu:
+----------------------------
+    Accounting > Payment Follow-Up / Send Email and letters
+
+It will generate a PDF / send emails / set manual actions according to the the
+different levels of recall defined. You can define different policies for
+different companies.
+
+Note that if you want to check the follow-up level for a given partner/account
+------------------------------------------------------------------------------
+entry, you can do from in the menu:
+-----------------------------------
+    Accounting > Reporting > Accounting > **Follow-ups Analysis
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-financial-tools/issues>`_. In case of trouble,
+please check there if your issue has already been reported. If you spotted it
+first, help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Vincent revaville <vincent.renaville@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Vicent Cubells <vicent.cubells@tecnativa.com>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```

Comment:
You need to add this disclaimer: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#backporting-odoo-modules